.github/workflows/main-kaoto.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
         sed -i '/kaoto-ui/d' package.json
         yarn
         cp -f package.json.bak package.json
-        yarn link kaoto-ui
+        yarn link @kaoto/kaoto-ui
```

Comment:
Now that @kaoto/kaoto-ui is published as an npm package we shouldn't have to do all the steps to remove the line with `kaoto-ui` as a dependency and then `yarn link` etc. We should only have to install `@kaoto-ui/kaoto-ui` and it will pull it in automatically.

Reply:
right, it is part of the PR to use the released version: https://github.com/KaotoIO/vscode-kaoto/pull/68